CHANGELOG.md

```diff
@@ -348,4 +349,4 @@ Changelogs were kept sporadically before this point
 
 - Major change to the way we store and cache users. Previously we cached users in-memory for 10 minutes before pushing any changes to the MySQL database. We have no removed this layer entirely, and changes made in the bot will be pushed immediately. This will greatly reduce the memory footprint of pajbot, but put a bigger strain on the database.
 - Moved a bunch of columns from MySQL to redis. (num_lines, last_seen, last_active, banned, ignored)
-- A bunch of more things that I don't remember, first changelog :sunglasses::ok_hand:
+- A bunch of more things that I don't remember, first changelog :sunglasses::ok_hand:
```

Comment:
this line probably should not be in the diff, is there some invisible characters or newlines added/removed here?

Reply:
👍